Userland/Libraries/LibMarkdown/HorizontalRule.h

```diff
@@ -21,7 +21,7 @@ class HorizontalRule final : public Block {
     }
     virtual ~HorizontalRule() override { }
 
-    virtual String render_to_html() const override;
+    virtual String render_to_html(bool tight = false) const override;
```

Comment:
Some of the overrides for this function have default values for `tight`, and some don't.
Please pick one and go with it?

Reply:
I went with all default values